src/components/Collection/CollectionList/CollectionList.tsx

```diff
@@ -1,76 +1,113 @@
+import { useState } from 'react';
+
 import useTranslation from 'next-translate/useTranslation';
+import Link from 'next/link';
+import useSWR from 'swr';
+
+import OverflowMenuIcon from '../../../../public/icons/menu_more_horiz.svg';
+import BookmarkIcon from '../../../../public/icons/unbookmarked.svg';
+import CollectionSorter from '../CollectionSorter/CollectionSorter';
 
 import styles from './CollectionList.module.scss';
+import DeleteCollectionAction from './DeleteCollectionAction';
+import RenameCollectionAction from './RenameCollectionAction';
 
-import Button, { ButtonShape, ButtonSize, ButtonVariant } from '@/dls/Button/Button';
-import PopoverMenu from '@/dls/PopoverMenu/PopoverMenu';
-import ChevronDownIcon from '@/icons/chevron-down.svg';
-import OverflowMenuIcon from '@/icons/menu_more_horiz.svg';
-import BookmarkIcon from '@/icons/unbookmarked.svg';
+import Button, { ButtonShape, ButtonSize, ButtonVariant } from 'src/components/dls/Button/Button';
+import PopoverMenu from 'src/components/dls/PopoverMenu/PopoverMenu';
+import { getCollectionsList } from 'src/utils/auth/api';
+import { makeCollectionsUrl } from 'src/utils/auth/apiPaths';
 
-type Collection = {
-  id: string | number;
-  name: string;
-  itemCount: number;
-};
+const defaultSortOptionId = 'recentlyUpdated';
```

Comment:
This should be using the enum instead of a string value. Also since it's fixed we should set it as `DEFAULT_SORT_OPTION`.